app.js

```diff
@@ -1,13 +1,42 @@
 'use strict';
 
 const Homey = require('homey');
+const SonyBraviaAndroidTvCommunicator = require('./helpers/sony-bravia-android-tv-communicator');
+const SonyBraviaFlowActions = require('../../definitions/flow-actions');
 
 class SonyBraviaAndroidTvApp extends Homey.App {
 
-	onInit() {
+	async onInit() {
 		this.log('SonyBraviaAndroidTvApp is running...');
+    await this.registerFlowListeners();
```

Comment:
Uhh ohhh indentation seems broken here 😢

Reply:
Could it be a 2 space vs 4 space indentation issue, could see the problem. Reformatted the documented just in case,